openpose/src/pose_estimation.py

```diff
@@ -8,13 +8,16 @@
 OPENPOSE_MODEL_DIR = os.environ["OPENPOSE_MODEL_DIR"]
 
 
-def perform_openpose_pose_estimation(input_path: str, options: dict):
+def perform_openpose_pose_estimation(input_path: str, options: dict, multi_person_detection: bool = False):
     video_capture = cv2.VideoCapture(input_path)
     video_height = int(video_capture.get(cv2.CAP_PROP_FRAME_HEIGHT))
 
-    params = prepare_openpose_params(options, video_height)
+    if multi_person_detection:
+        params = prepare_openpose_params(options, video_height, unknown_maximum_number_of_people=True)
+    else:
+        params = prepare_openpose_params(options, video_height) # default behaviour for maskanyone
```

Comment:
```suggestion
    params = prepare_openpose_params(options, video_height, multi_person_detection)
```
You can simply pass the bool to the function without needing the if-else.

Reply:
good suggestion